sdk/core/Azure.Core/src/Azure.Core.props

```diff
@@ -1,15 +1,11 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
```

Comment:
Do we want to split them completely and only update AppConfig every time Azure.Core ships?
If we want to test AppConfig with new Azure.Core we could create some internal builds and pass `UseProjectReferenceToAzureBase=false`

Reply:
We might revert to using P2Ps after preview2 of AppConfig ships and use P2Ps while they both are in preview.

We would need a plan on how to manage this long term.

Reply:
I would prefer not deleting the code. We could comment it out or pass it to our build

Reply:
It would have to be moved into the AppConfig project when brought back. We should stop setting this property globally.